lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -360,12 +360,8 @@ function paraneue_dosomething_get_themed_reportback_item($data) {
     return NULL;
   }
 
-  if ($data->id && !$data->disable_reactions) {
-    $data->reaction_totals = [
-      'heart' => dosomething_kudos_get_total_for_file_by_term_name($data->id, 'heart'),
-    ];
-
-    $data->existing_kids = dosomething_kudos_get_kudos_for_user_by_file($data->id);
+  if (! dosomething_kudos_disable_reactions($data->campaign->id) && $data->id) {
```

Comment:
Bit picky, but I would put the `$data->id` first, because if that doesn't exist then there's no point in calling the function to check if kudos reactions are disabled. If the first item in an `&&` statement is false, then the whole thing is false and the second check doesn't even bother happening.